mesonbuild/interpreter.py

```diff
@@ -664,6 +664,7 @@ def __init__(self, compiler, env):
                              'cmd_array': self.cmd_array_method,
                              'find_library': self.find_library_method,
                              'has_argument': self.has_argument_method,
+                             'has_func_attribute': self.has_func_attribute_method,
```

Comment:
I think this should be `has_function_attribute`. Below `supported` and `arguments` is not abbreviated, and this is not going to be a very common option, so no need to economize.

Reply:
sure.